velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -86,6 +86,39 @@ class HiveConnectorTestBase : public OperatorTestBase {
         remainingFilter);
   }
 
+  /// @param targetDirectory Final directory of the target table after commit.
+  /// @param writeDirectory Write directory of the target table before commit.
+  /// @param tableType Whether to create a new table, insert into an existing
+  /// table, or write a temporary table.
+  /// @param writeMode How to write to the target directory.
+  static std::shared_ptr<connector::hive::LocationHandle> makeLocationHandle(
+      std::string targetDirectory,
+      std::optional<std::string> writeDirectory = std::nullopt,
```

Comment:
```
const std::optional<std::string>& writeDirectory
```
How about we call it stagingDirectory?

Reply:
Would be better to keep the naming consistent w/ protocol::LocationHandle

Reply:
I don't think it makes sense to aim for consistency with one specific Velox application. Velox is a generic library intended to be used in multiple systems / applications, hence, it would be great for the APIs to make sense independent of any specific application.